recipe/meta.yaml

```diff
@@ -1,71 +1,129 @@
-{% set version = "1.2.2" %}
+{% set version = "1.3.1" %}
+{% set build_number = "0" %}
 
 package:
-  name: basemap
+  name: basemap-split
   version: {{ version }}
 
 source:
-  url: https://github.com/matplotlib/basemap/archive/v{{ version }}rel.tar.gz
-  sha256: 7e6ee5d03b10168862cff82bfa819df8264c04f078eac4549a22dd2631696613
+  url: https://github.com/matplotlib/basemap/archive/v{{ version }}.tar.gz
+  sha256: 8f4ed7d5736711ba2b413e5fe038d4349d35eb3f28cba836094b6cca5f0a634f
   patches:
-    - data.patch
+    # Update messages regarding installation of `basemap-data-hires`.
+    - install_hires_instructions.patch
 
 build:
-  number: 2
-  # This package isn't included to the SOW Packages list
-  skip: True  # [linux and s390x]
-  skip: True  # [win and py2k]
-  ignore_run_exports:
-    - proj
+  number: {{ build_number }}
+  skip: True  # [s390x]  Missing dependencies.
+  skip: True  # [win32]  Missing dependencies in win32 arch and not expecting they will be updated.
+  skip: True  # [py<3.8]  Dependencies require 3.8+
 
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
     - m2-patch  # [win]
     - patch  # [not win]
-  host:
-    - python
-    - cython
-    - geos
-    - numpy
-    - pip
-    - proj
-    - pyproj
-    - setuptools
-    - wheel
-  run:
-    - python
-    - {{ pin_compatible('numpy') }}
-    - geos
-    - matplotlib-base >=1.0.0,!=3.0.1
-    - proj
-    - pyproj >=1.9.3
-    - pyshp >=1.2.0
-    - six
 
-test:
-  imports:
-    - mpl_toolkits.basemap
-  requires:
-    - pip
-  commands:
-    - pip check
-    - python -c "from mpl_toolkits.basemap import Basemap"
 
-about:
-  home: http://matplotlib.org/basemap
-  license: MIT
-  license_family: MIT
-  license_file: README.md
-  summary: 'Plot on map projections using matplotlib'
-  description: |
-    Basemap toolkit is a library for plotting 2D data on maps in Python.
-    It is similar in functionality to the matlab mapping toolbox, the IDL mapping facilities, GrADS, or the Generic Mapping Tools.
-  dev_url: https://github.com/matplotlib/basemap
-  doc_url: http://matplotlib.org/basemap/
-  doc_source_url: https://github.com/matplotlib/basemap/blob/master/doc/index.rst
+outputs:
+
+  - name: basemap
+    script: build-basemap.sh  # [not win]
+    script: bld-basemap.bat   # [win]
+    build:
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - python
+        - pip
+        - cython >=0.29,<3.1
+        - numpy >=1.21  # [not ppc64le]
+        - numpy >=1.19  # [ppc64le]
+        - geos
+        - pyproj
+        - setuptools
+        - wheel
+      run:
+        - {{ pin_subpackage('basemap-data') }}
+        - python
+        - {{ pin_compatible('numpy') }}
+        - {{ pin_compatible('geos') }}
+        - matplotlib-base >=1.5,!=3.0.1
+        - pyproj >=1.9.3
```

Comment:
pyproj for v1.3.1 https://github.com/matplotlib/basemap/blob/9389196d81dc44809537e8196d8e7cb87e6bb1ef/packages/basemap/requirements.txt#L32
But it can be better to update to v1.3.2